students/jjakub/session03/mailroom.py

```diff
@@ -39,35 +43,50 @@ def donor_rpt():
     donor_db_sum = {donor: sum(donor_db[donor]) for donor in donor_db}
     key_sort = sorted(donor_db_sum, key=donor_db_sum.__getitem__,reverse=True)
     for donor in key_sort:
-        print(f'{donor.capitalize():13}', f'{sum(donor_db.get(donor)):>10.2f}', f'{len(donor_db.get(donor)):>10}', f'${sum(donor_db.get(donor))/len(donor_db.get(donor)):>10.2f}')
+        print(f'{donor.capitalize():13}',\
+            f'{sum(donor_db.get(donor)):>10.2f}',\
+            f'{len(donor_db.get(donor)):>10}',\
+            f'${sum(donor_db.get(donor))/len(donor_db.get(donor)):>10.2f}'
+        )
```

Comment:
Much more readable by breaking a long line into multiple lines. Just one minor thing is that you don't have to put `\` at the end of a line (that's to be continued further) if it's within a parenthesis pair (like in this case).